templates/emails/collective.expense.missing.receipt.hbs

```diff
@@ -11,7 +11,7 @@ Subject: Action Required: Submit receipt for card expense
 <p>
 {{#if user}}
 A card attached to {{fromCollective.name}} was charged {{currency expense.amount currency=expense.currency}} and we need the responsible admin to input additional
-information.
+information. This expense was submitted to {{collective.name}}.
```

Comment:
Would be great for this to be a link, and same for `fromCollective`

Reply:
Done. 👍🏽